job-server/src/main/resources/reference.conf

```diff
@@ -0,0 +1,6 @@
+spark {
+  jobserver {
+    # Maximum number of dispatcher threads for each Spark context
+    max-dispatchers-per-context = 2
```

Comment:
The reason for both is the same; Spark has its own threads for running tasks and we only use these threads to dispatch them to Spark for running. Hence the name "dispatcher" and low number of threads.

A question we may ask is; since there are two threads but not a unique one, how will Spark react when we call setJobGroup. Then, if our threads are A, and B and the invocation is carried out by A; then only and all jobs dispatched by A would be given the specific id. We should perhaps reduce this to a single thread executor. What do you think?